<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #0A0A0A;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Sprites/back.png</normaloff>Sprites/back.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>70</y>
     <width>160</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="theme_label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;</string>
      </property>
      <property name="text">
       <string>Темы</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="light_theme">
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;</string>
      </property>
      <property name="text">
       <string>Светлая тема</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="dark_theme">
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;</string>
      </property>
      <property name="text">
       <string>Тёмная тема</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>dark_theme</zorder>
   <zorder>light_theme</zorder>
   <zorder>theme_label</zorder>
  </widget>
  <widget class="QPushButton" name="password_button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>280</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>Блокировать</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Sprites/lock.png</normaloff>Sprites/lock.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="delpassword_button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>280</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>Разблокировать</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Sprites/open_lock.png</normaloff>Sprites/open_lock.png</iconset>
   </property>
  </widget>
  <widget class="QCheckBox" name="autosave_btn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>360</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Автосохранение</string>
   </property>
  </widget>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>delpassword_button</zorder>
  <zorder>password_button</zorder>
  <zorder>frame</zorder>
  <zorder>autosave_btn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
